src/RootComponent.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Relay from 'react-relay';
+
+import Container from './Container';
+import RouteAggregator from './RouteAggregator';
+
+export default class RootComponent extends React.Component {
+  static displayName = 'ReactRouterRelay.RootComponent';
+
+  static propTypes = {
+    routerProps: React.PropTypes.object.isRequired,
+  };
+
+  static childContextTypes = {
+    routeAggregator: React.PropTypes.instanceOf(RouteAggregator).isRequired,
+  };
+
+  constructor(props, context) {
+    super(props, context);
+
+    this.routeAggregator = new RouteAggregator();
```

Comment:
Even on React components? Naively it seems like it shouldn't even be possible to get a reference to the `RootComponent` element in order to access this property.